woocommerce/woocommerce-gutenberg-products-block
# Revise Storybook entries for Checkout Blocks/totals components

Revise Storybook entries for Checkout Blocks/totals components


This PR improves the stories for: Checkout Blocks > Totals: Fees, Item, Subtotal & Taxes.

It adds Storybook controls to change dynamically props of these components.


Fixes #5432 

### Screenshots
|Before|After|
|-|-|
|![WooCommerce_Blocks___Checkout_Blocks___totals___Fees_-_Default_⋅_Storybook](https://user-images.githubusercontent.com/905781/154076843-c5eb0fee-101b-4465-9cb4-c1783a56cab6.jpg)|![WooCommerce_Blocks___Checkout_Blocks___totals___Fees_-_Default_⋅_Storybook](https://user-images.githubusercontent.com/905781/154076286-d41cbd45-2f48-4478-8032-79709006c46f.jpg)|

### Testing

### Manual Testing
How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Explore the following stories:
- [Fees](http://localhost:6006/?path=/story/woocommerce-blocks-checkout-blocks-totals-fees--default)
- [Item](http://localhost:6006/?path=/story/woocommerce-blocks-checkout-blocks-totals-item--default)
- [Subtotal](http://localhost:6006/?path=/story/woocommerce-blocks-checkout-blocks-totals-subtotal--default)
- [Taxes](http://localhost:6006/?path=/story/woocommerce-blocks-checkout-blocks-totals-taxes--default)

3. Please make sure all the controls are working correctly
4. Please make sure there are no Storybook errors
5. There are 2 outstanding issues, which I believe we can ignore for now:
- **Item > description** control is highlighted in red (error), due to HTML markup in the string of the description - I have searched but didn't find a great solution for this
- **Taxes > showRateAfterTaxName** - doesn't work due to being dependent upon a value coming from outside of the component

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).


Co-authored-by: gigitux <gigitux@gmail.com>

#### Automated Tests
